### ISSUE TITLE
Can't move the window!

### ISSUE THREAD

**Issue by ZeroCool22** [Si13n7/CoD2MPWindowed#2]
It's working, but i would like to can **move the window** of the game to my liking.

https://user-images.githubusercontent.com/13344308/227749929-aacefb3d-12df-4ac4-99e8-2be22228ed0f.mp4

**Comment by ghost** [comment]
Had the same issue using xtnded/iw2clientdll

**Comment by ZeroCool22** [comment]
> Had the same issue using xtnded/iw2clientdll

You found a fix?

**Comment by ghost** [comment]
> > Had the same issue using xtnded/iw2clientdll
> 
> You found a fix?

No i didn't search for and don't use cod2 anymore

**Comment by ZeroCool22** [comment]
> > > Had the same issue using xtnded/iw2clientdll
> > 
> > 
> > You found a fix?
> 
> No i didn't search for and don't use cod2 anymore

What you playing now?

**Comment by ZeroCool22** [comment]
Fixed with the last version, thx!

https://github.com/user-attachments/assets/127fe74a-81fd-43b4-8cbf-420d3d3261c4

**Comment by ghost** [comment]
Fixed with last version? O_o
Last release is from Feb 16, 2019 and your Issue from Mar 26, 2023 so i don't understand but it's cool if it's fixed :)
I play cod1 btw (version 1.1)

**Comment by koryboc** [comment]
Hello everyone! I doubt it will work on Windows 11. This tool patches the game files at runtime to enable windowed mode in multiplayer. While it should work everywhere, the game itself does not allow the window to be moved even in windowed mode. Therefore, this tool additionally creates its own window into which the game window is embedded, creating the illusion that this is the game window that you can move without issues. Unfortunately, this method no longer works on Windows 11. It is no longer possible to manipulate the game window in this way. So this tool does not support Windows 11, and I personally will probably never support it either, as I no longer use Windows myself.

**Comment by ghost** [comment]
Thanks for the info @Si13n7 
Do you play on Linux now or you don't play these days?

**Comment by koryboc** [comment]
> Thanks for the info @Si13n7 Do you play on Linux now or you don't play these days?

Yes, I use Linux — specifically Manjaro with KDE Plasma, which is based on Arch. In my opinion, it's one of the best distributions, especially for gamers. I still play games, although not as much as I used to. Most games run as well on my setup as they do on Windows, and some even perform significantly better. However, there are a few games that don’t work on Linux, such as Valorant, mainly due to its anti-cheat system. I'm not a fan of such systems anyway; they dig too deep into the operating system and ultimately don't provide much benefit. Therefore, I wouldn’t play those kinds of games anymore. Personally, I can play everything I want on Linux.

**Comment by ghost** [comment]
Interesting info :) I would be curious to try cod(1) with Linux someday, although i don't really see the purpose
Seems to me video games are made to run on Windows, and they run better on it, my pc has a dual boot Debian+Windows, the Windows was originally meant only to play, but i feel it more pleasant and convenient to use than Linux anyway
If i was concerned about my data/privacy, i would use only Linux when not playing

**Comment by ghost** [comment]
ps: i can still find purposes e.g. very slow pc, don't want to dualboot...

**Comment by koryboc** [comment]
> Seems to me video games are made to run on Windows, and they run better on it, my pc has a dual boot Debian+Windows, the Windows was originally meant only to play, but i feel it more pleasant and convenient to use than Linux anyway

Debian is one of the worst choices for desktop use, especially if you rely on recent developments. In the Linux world, there are constantly new updates and improvements: the Linux kernel receives an update every nine weeks, often bringing performance enhancements, and other software packages are continuously developed as well. New features are added, and bugs are fixed—things that you barely get to experience with Debian.

Every two years, Debian freezes a "snapshot" of the current software, and from that point on, only security updates are applied. If a program is buggy at the time of this snapshot—for example, if the user interface frequently crashes or has other severe bugs—these issues will persist for the entire lifespan of that Debian version. A few years ago, KDE Plasma was an example of this: in Debian (and, as a result, in distributions based on it like Ubuntu and Mint), it was extremely buggy. Although the developers fixed the issues within a few days, Debian users had to live with these bugs for two years.

Debian is particularly unsuitable for gamers, as performance improvements in the kernel and optimized graphics drivers are crucial and need to be updated regularly. In recent years, a lot has happened in terms of Linux optimizations for gaming, and those who rely on an outdated distribution miss out on key developments. For these reasons, I wouldn't recommend Debian for desktop users, especially not for gamers. I’ve noticed over the past year a whole list of games where I get higher FPS on Linux now, even though they were developed for Windows. Years ago, I could never have imagined this would happen. Now, I think this is just the beginning.

Personally, I prefer Arch, though I can also recommend Fedora as a desktop distribution. For desktop users, I would advise against other distributions, and with Debian, I would even issue a clear warning. With over 20 years of experience in the Linux field, I consider Debian too outdated for everyday desktop use.

While I appreciate Debian’s focus on security, and the security vulnerabilities it identifies ultimately benefit other distributions as well, the Linux sector evolves too quickly for Debian to keep up. After a year, a Debian system feels like it’s stuck at the level of Windows 2000, while Arch has reached a state more akin to Windows 7. This makes Debian particularly unsuitable for gamers, who rely on regular performance updates and the latest optimizations.

**Comment by koryboc** [comment]
Please don’t get me wrong, my goal is not to convince you to switch to Linux. Rather, I just want to share my opinion on Debian, Arch, and gaming on Linux. Everyone should decide for themselves which operating system best suits their needs, as Linux also has its weaknesses. One of the biggest, paradoxically, is its diversity—it’s both a strength and a challenge. For newcomers, this diversity often means that getting started with Linux can be a rocky road.

Even choosing the right desktop environment can shape your entire experience with Linux, and the issue continues across every software category. For many use cases, there are numerous programs that achieve similar goals, but few of them meet personal requirements perfectly—and finding these can be quite tedious at first. It’s completely fine if you just don’t feel like dealing with this.

**Comment by ghost** [comment]
Thank you very much for these great info, that's not the first time i hear positive about Arch
These days I don't really need Linux, but when I do I'm used to take Debian because it seems like easier to setup than the others, i also had some issues with Ubuntu that i didn't get with Debian, so it gives me some feeling of reliability/strength

When I need an IDE i take xfce, the offered Gnome in Debian is really messy imo, full of stuff like games etc 🤮
i'm not interested about fancy UI when using Linux

I would for sure try Arch if i want to invest myself into Linux someday

I heard about some games offering more fps in Linux than Windows, that's a reason i would be curious to try cod1 in Linux sometimes
